ui-events-web/CHANGELOG.md

```diff
@@ -8,16 +8,12 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 # Changelog
 
-The latest published UI Events for Web release is [0.1.0](#010-2025-XX-XX) which was released on 2025-XX-XX.
-You can find its changes [documented below](#010-2025-XX-XX).
-
```

Comment:
This removal is not needed.This removal is not needed since we aren't publishing it yet.